source/SpinalHDL/Data types/bits.rst

```diff
@@ -282,11 +307,15 @@ Misc
      - Return the index of the MSB (highest allowed index for Bits)
      - Int
    * - x.msb
-     - Return the most significant bit
+     - Return the most significant bit value
```

Comment:
I find this one to be more confusing with the change, "most significant bit" is a well defined term, if you add "value" I get suspicious...

Reply:
Interesing: to me, if you ask for the "the most significant bit" you ask for the bit, not it's position. - probably an indication that we should make this clear.

I'm a bit rusty on my english and not sure if that should then be "the most significant bit value" or "the most significant bits value", maybe go for "the value of the most significant bit"?